build_tools/cmake/iree_cc_test.cmake

```diff
@@ -72,6 +72,12 @@ function(iree_cc_test)
   set(_NAME "${_PACKAGE_NAME}_${_RULE_NAME}")
 
   add_executable(${_NAME} "")
+  add_executable(${_RULE_NAME} ALIAS ${_NAME})
+  if(_RULE_OUT)
```

Comment:
This just makes it consistent - if we want to drop it from cc_binary I'd rather do that separately.

Reply:
Well currently it doesn't do anything because _RULE_OUT isn't parsed anywhere (and cmake is of course totally fine with you using undefined variables :roll_eyes:). So to actually make it consistent you need to add that parsing. I agree that dropping it from cc_binary is totally separate from this change. I'm just saying I'd like to do that anyways, so I'd just drop this from the change, since making it consistent is also basically unrelated to the rest of this change.

Reply:
Seems this was merged as is. It's currently just dead code. I will send a PR to delete this in cc_test and cc_binary